packages/babel-plugin-transform-async-to-promises/src/index.js

```diff
@@ -0,0 +1,113 @@
+/* Babel 7 transform */
+
+const babylon = require("babylon");
```

Comment:
This dep doesn’t seem to be listed in package.json

Reply:
As above, fixed in https://github.com/babel/babel/pull/7076/commits/943a44c522dca0acaabf3d1b73317ab155124f99